chap-architecture.tex

```diff
@@ -36,8 +36,9 @@ \section{Architectural Instantiations of CHERI Protection}
   implementations, adaptations of our CheriBSD and CheriFreeRTOS operating
   systems, Clang/LLVM/LLD toolchain, GDB debugger, and application suite.
 
-  We expect continuing disruptive modification to this ISA mapping, including
-  reencoding of many key instructions, as it transitions to a more mature
+  We aim to propose 64-bit CHERI-RISC-V as a RISC-V extension with
+  minimal adjustments.  We consider 32-bit CHERI-RISC-V less mature
+  and expect future disruptive modifications as it transitions to a more mature
```

Comment:
I’m good with committing this text, but it would be worth thinking about whether an example or two here would help explain why we think that -- e.g., that the capability encoding would ideally provide more bounds precision, and perhaps invest fewer bits in permissions.

Reply:
I have added a small paragraph in the RISC-V chapter that tries to explain this a bit further including the specific examples you gave.